libs/langchain/langchain/llms/bedrock.py

```diff
@@ -26,7 +27,21 @@ def prepare_input(
         cls, provider: str, prompt: str, model_kwargs: Dict[str, Any]
     ) -> Dict[str, Any]:
         input_body = {**model_kwargs}
-        if provider == "anthropic" or provider == "ai21":
+        if provider == "anthropic":
+            if prompt.startswith("Human:") and prompt.endswith("Assistant:"):
```

Comment:
Ideally these should come from the `anthropic` runtime. See here.
https://github.com/langchain-ai/langchain/blob/master/libs/langchain/langchain/llms/anthropic.py#L92-L93

Also, this seems a bit different from Anthropic has, is this intended?
https://github.com/langchain-ai/langchain/blob/master/libs/langchain/langchain/llms/anthropic.py#L186-L187